.github/workflows/scorecard.yml

```diff
@@ -13,6 +13,8 @@ on:
     - cron: '27 22 * * 4'
   push:
     branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```

Comment:
Problem to introduce this in PRs is that we are going to introduce the red failure check that might be happening for every PR, even if its not introduced in the corresponding PR (thus pushing that bad habit of making devs to ignore the CI checks that we wanted to fix initially in our pipelines).

Reply:
removed pr trigger